src/actions/refresh-db/index.ts

```diff
@@ -53,7 +53,7 @@ export const refreshDb: RefreshDbFn = async () => {
       const courseId = course.appxCourseId.toString();
       const data = await checkUserEmailForPurchase(email, courseId);
 
-      if (data.data === '1') {
+      if (data) {
```

Comment:
why did u update this?
This will lead to every user having access to every course

Reply:
U r right...at first it did give access to all users but eventually it didn't after testing it multiple times...so I reduced the code to this...